.github/workflows/manual_msi_build.yml

```diff
@@ -0,0 +1,89 @@
+name: manual_msi_build
+on:
+  workflow_dispatch:
```

Comment:
can we add a parameter to specify the branch/commit? Assuming this only works from main when merged?

Reply:
@cpheps if you look at that link in the summary, i think it lets you select the branch you want to use